### PR TITLE
side menu navigation

### PR DESCRIPTION
**Pull request by sandikbarr**
#107

### CODE REVIEW

**Comment by sandikbarr** [comment]
I'm having a bit of trouble finding what is going on that causes the content area to get slightly larger when the summary table is expanded. It doesn't seem to be the negative margins on the table wrapper (for the 3px table border.) The Chrome dev tools calculated properties don't seem to show it any larger. If I can't figure it out by EOD today, I'm just gonna create a ticket for it.

File: src/app/downloads/downloads.routing.ts [hunk 1]
```diff
@@ -5,6 +5,21 @@ import { AuthGuard, DeactivateGuard } from 'ngx-prx-styleguide';
 import { DownloadsComponent } from './downloads.component';
 
 export const downloadsRoutes: Routes = [
+  {
+    path: ':seriesId',
+    pathMatch: 'full',
+    redirectTo: ':seriesId/downloads/podcast/daily'
```
Comment (gcampo88) [inline src/app/downloads/downloads.routing.ts]: very nice!

File: src/app/geo/geo.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }).compileComponents().then(() => {
+      fix = TestBed.createComponent(GeoComponent);
+      comp = fix.componentInstance;
+      fix.detectChanges();
+      de = fix.debugElement;
+      el = de.nativeElement;
+    });
+  }));
+
+  it('should create the component', async(() => {
```
Comment (gcampo88) [inline src/app/geo/geo.component.spec.ts]: 🌮  for setting up the testing stuff

File: src/app/shared/nav/nav-menu.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      path: ':seriesId/traffic-sources',
+      component: TestComponent
+    }
+  ];
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        NavMenuComponent,
+        NavMenuComponent,
```
Comment (gcampo88) [inline src/app/shared/nav/nav-menu.component.spec.ts]: dup line w/ above

**Pull request merged**